README.rst

```diff
@@ -20,33 +20,238 @@
+
+In addition there are a set of curated teaching notebooks located in
+``landlab/notebooks/teaching``.
+
+What License does Landlab use?
+------------------------------
+
+MIT (see the file LICENSE.txt)
+
+I used Landlab and want to cite it. How do I do this correctly?
+---------------------------------------------------------------
+
+The following reference refers to the entire Landlab package.
+
+`Hobley, D. E. J. <https://www.earth-surf-dynam.net/5/21/2017/>`__, Adams,
+J. M., Nudurupati, S. S., Hutton, E. W. H., Gasparini, N. M., Istanbulluoglu,
+E. and Tucker, G. E., 2017, Creative computing with Landlab: an open-source
+toolkit for building, coupling, and exploring two-dimensional numerical models
+of Earth-surface dynamics, Earth Surface Dynamics, 5, p 21-46,
+10.5194/esurf-5-21-2017.
+
+BibTeX format:
+::
+
+ @article{Hobley2017,
+          Author = {Hobley, D. E. J. and Adams, J. M. and
+                    Nudurupati, S. S. and Hutton, E. W. H.
+                    and Gasparini, N. M. and Istanbulluoglu,
+                    E. and Tucker, G. E.},
+          Journal = {Earth Surface Dynamics},
+          Year = {2017},
+          Title = {Creative computing with Landlab: an open-source
+                   toolkit for building, coupling, and exploring
+                   two-dimensional numerical models of
+                   Earth-surface dynamics},
+          Number = {5},
+          Pages = {21-46},
+          Doi = {10.5194/esurf-5-21-2017}}
+
+
+In addition, depending on what parts of Landlab you use, you may need to cite
+component-specific. Refer to the References section of each component and
+`this page <>`_ which compiles component references.
+
+I think I found a bug. What should I do?
+----------------------------------------
+
+Please make an Issue describing the bug so we can address it, or work with you
+to address it. Please try to provide a
+`minimal, reproducible example <https://stackoverflow.com/help/minimal-reproducible-example>`_.
+
+I found something in the documentation that isn't clear. What should I do?
+--------------------------------------------------------------------------
+
+Please make an Issue describing the what isn't clear to you. Someone will tag
+the most appropriate member of the core Landlab team. We will work to clarify
+your question and revise the documentation so that it is clear for the next user.
+
+I'm interested in contributing to Landlab. Where do I get started?
+------------------------------------------------------------------
+
+Thank you for your interest! Refer to ``CONTRIBUTING.md`` and
+`this <https://landlab.readthedocs.io/en/master/development/index.html#development>`_
+page in the documentation that describes contribution guidelines.
+
+How is the Landlab package structured?
+--------------------------------------
+
+The `following page <>` in the documentation describes the package structure. 
```

Comment:
not sure of the link

Reply:
Yeah. These few empty links are things that will get put into the docs by #1117...